tests/robot-tests/tests/libs/admin-common.robot

```diff
@@ -73,9 +73,9 @@ user selects dashboard theme and topic if possible
     [Arguments]
```

Comment:
Would also suggest changing the final lines of `user creates public prerelease access list` and `user updates public prerelease access list` to:

```
    user clicks button    Save access list
    user waits until element contains    css:[data-testid="publicPreReleaseAccessListPreview"]
    ...    ${content}
```

as the previous waiting after the button click was but this seems to be correct.

Reply:
I've actually included this in the 4448 PR as I was having a lot of failures around it, so no need to change it here.